modules/tinymce/src/plugins/autoresize/test/ts/browser/AutoresizePluginDocumentTest.ts

```diff
@@ -0,0 +1,78 @@
+import { ApproxStructure, Assertions, Waiter } from '@ephox/agar';
+import { beforeEach, context, describe, it } from '@ephox/bedrock-client';
+import { Cell } from '@ephox/katamari';
+import { TinyHooks, TinyUiActions } from '@ephox/wrap-mcagar';
+import { assert } from 'chai';
+
+import Editor from 'tinymce/core/api/Editor';
+import AutoresizePlugin from 'tinymce/plugins/autoresize/Plugin';
+import FullscreenPlugin from 'tinymce/plugins/fullscreen/Plugin';
+
+describe('browser.tinymce.plugins.autoresize.AutoresizePluginTest', () => {
+  const assertEditorHeightAbove = (editor: Editor, minHeight: number) => {
+    const editorHeight = editor.getContainer().offsetHeight;
+    assert.isAtLeast(editorHeight, minHeight, `editor height should be above: ${editorHeight}>=${minHeight}`);
+  };
+
+  const assertEditorContentApproxHeight = (editor: Editor, height: number, diff: number = 5) => {
+    // Get the editor height, but exclude the 10px margin from the calculations
+    const editorContentHeight = editor.getContentAreaContainer().offsetHeight - 10;
+    assert.isAtMost(Math.abs(editorContentHeight - height), diff, `editor content height should be approx (within ${diff}px): ${editorContentHeight} ~= ${height}`);
+  };
+
+  context('common tests', () => {
+    const resizeEventsCount = Cell(0);
+    const hook = TinyHooks.bddSetup<Editor>({
+      plugins: 'autoresize fullscreen',
+      menubar: false,
+      toolbar: 'autoresize',
+      base_url: '/project/tinymce/js/tinymce',
+      autoresize_bottom_margin: 3000,
+      content_css: 'document',
+      toolbar_mode: 'floating',
+      // Override the content css margins, so they don't come into play
+      content_style: 'body { margin: 0; margin-top: 10px; }',
+      setup: (editor: Editor) => {
+        editor.on('ResizeEditor', () => {
+          resizeEventsCount.set(resizeEventsCount.get() + 1);
+        });
+      }
+    }, [ AutoresizePlugin, FullscreenPlugin ], true);
+
+    beforeEach(() => {
+      resizeEventsCount.set(0);
+    });
+
+    it('TBA: Should not have a resize handle visible by default', async () => {
```

Comment:
I think these TBA is is just annoying if they never get changed to anything then why have them. Either use the ticket ID here too or just remove the `TBA:` I never seen anyone ever change from that value.

Reply:
Wups, forgot to change those.

Reply:
Originally these were testrail case numbers, and QA would come through later to make sure automated tests were tracked in testrail (so we didn't manually test something that was covered by automated testing).

Some time later, whether intentionally or just because it made sense, we started using JIRA ticket numbers - and we rarely make PRs without JIRA tickets now.